coderd/workspaceagents.go

```diff
@@ -138,6 +146,14 @@ func (api *API) workspaceAgentMetadata(rw http.ResponseWriter, r *http.Request)
 	})
 }
 
+// @Summary Submit workspace agent version
+// @ID submit-workspace-workspace-agent-version
+// @Security CoderSessionToken
+// @Produce application/json
+// @Tags Agents
+// @Param request body codersdk.PostWorkspaceAgentVersionRequest true "Version request"
+// @Success 200
+// @Router /workspaceagents/me/version [post]
```

Comment:
Hmm... the agent uses this route so the dashboard knows its version?

I think documenting these routes (routes that are not consumed by any clients but simply for agent<->dashboard communication) may be a bit confusing. Is there a way to add a description to`agent.md` or simply not generate docs for this?

Reply:
Fixed. 
I introduced an extension to swagger (`x-apidocgen`), which lets us skip specific operations.